tasks/getval.json

```diff
@@ -3,6 +3,10 @@
     "input_method": "stdin",
     "files": ["python_task_helper/files/task_helper.py"],
     "parameters": {
+      "hostname": {
```

Comment:
Hi
There's example and parameters in README that could be extended with the new option.